provider/pkg/provider/provider.go

```diff
@@ -383,12 +394,38 @@ func (p *cfnProvider) Configure(ctx context.Context, req *pulumirpc.ConfigureReq
 		p.roleArn = &roleArn
 	}
 
-	if maxRetries, ok := vars["aws-native:config:maxRetries"]; ok && cfg.Retryer == nil {
-		num, err := strconv.Atoi(maxRetries)
+	maxRetries := 25
```

Comment:
Do I understand correctly that 25 didn't work for you? I thought you got through to the next error (related to `maxRetryRateTokens`).

I don't have crisp reasoning for 25 vs 1000000 other than a vague "25 should be enough for most users" and the fact that I stole "25" from another relevant open source project integrating with the same API. @kpitzen @thomas11 curious what you think?

Reply:
IMO infinite retries on rate limiting exceptions should be the default, and users can use this config value to reduce the threshold if needed.

Reply:
I think my only concerns around retrying infinitely is that it may not be clear to the user that's what's actually happening (especially in CI). Can we call out when retries occur / document clearly these settings and what their defaults imply? That is, by using this provider out of the box, we will retry infinitely, and that's a potential source of hangs?